common/web/types/src/package/kps-file.ts

```diff
@@ -64,8 +66,11 @@ export interface KpsFileContentFiles {
 
 export interface KpsFileContentFile {
   name: string;
+  /** @deprecated */
   description: string;
+  /** @deprecated */
   copyLocation: string;
+  /** @deprecated */
   fileType: string;
```

Comment:
Took the opportunity to mark these as deprecated; we really only care about the filename, the other details are never used any longer.